adoc/chapters/device_compiler.adoc

```diff
@@ -216,132 +216,31 @@ The restriction waiver in <<discarded-statement>> or
+  [code]#unsigned long long int#, [code]#float# and [code]#double#);
 
+- The fixed width types defined by the {cpp} core language (e.g.,
+  [code]#int8_t#, [code]#int16_t#, [code]#int32_t#, [code]#int64_t#); and
 
-[[table.types.fundamental]]
-.Fundamental data types supported by SYCL
-[width="100%",options="header",separator="@",cols="65%,35%"]
-|====
-@ Fundamental data type @ Description
-a@
-[source]
-----
-bool
-----
-   a@ A conditional data type which can be either true or false. The value
-      true expands to the integer constant 1 and the value false expands to the
-      integer constant 0.
-
-a@
-[source]
-----
-char
-----
-   a@ A signed or unsigned 8-bit integer, as defined by the {cpp} core language
-
-a@
-[source]
-----
-signed char
-----
-   a@ A signed 8-bit integer, as defined by the {cpp} core language
-
-a@
-[source]
-----
-unsigned char
-----
-   a@ An unsigned 8-bit integer, as defined by the {cpp} core language
-
-a@
-[source]
-----
-short int
-----
-   a@ A signed integer of at least 16-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-unsigned short int
-----
-   a@ An unsigned integer of at least 16-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-int
-----
-   a@ A signed integer of at least 16-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-unsigned int
-----
-   a@ An unsigned integer of at least 16-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-long int
-----
-   a@ A signed integer of at least 32-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-unsigned long int
-----
-   a@ An unsigned integer of at least 32-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-long long int
-----
-   a@ An integer of at least 64-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-unsigned long long int
-----
-   a@ An unsigned integer of at least 64-bits, as defined by the {cpp} core language
-
-a@
-[source]
-----
-float
-----
-   a@ A 32-bit floating-point. The float data type must conform to the IEEE 754
-      single precision storage format.
-
-a@
-[source]
-----
-double
-----
-   a@ A 64-bit floating-point. The double data type must conform to the IEEE 754
-      double precision storage format.  This type is only supported on devices
-      that have [code]#aspect::fp64#.
-
-|====
+- Any type aliases defined by the {cpp} core language (e.g.,
+  [code]#std::size_t#, [code]#std::ptrdiff_t#).
 
+All types which are available in device code must have the same size, alignment
+requirements, and representation on both host and device.
+A device compiler may rely on additional command-line options to provide this
+guarantee for specific combinations of host and device.
 
+The availability of these types in device code does not guarantee that they will
+be supported by all devices.
+Some types are only supported on devices with specific device aspects, as
+descrived in <<sec:device-aspects>>.
```

Comment:
```suggestion
The availability of these types in device code does not guarantee that they will
be supported by all devices.
Some types are only supported on devices with specific device aspects, as
described in <<sec:device-aspects>>.

Typo fix; this looks good otherwise.
```

Reply:
Good catch, thanks!  Fixed in d3238a4e649d6aad6ddc15a38f0820efcae5a762.